models/package.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function defineUserModel(sequelize, DataTypes) {
+  const Package = sequelize.define('Package', {
+    email: DataTypes.STRING,
+    trackingNumber: DataTypes.STRING,
+    carrier: DataTypes.STRING,
+  });
+
+  Package.createPackage = function createPackage(email, trackingNumber, carrier) {
+    return Package.create({
+      email,
+      trackingNumber,
+      carrier,
+    });
+  };
+
+  Package.findByEmail = function findByEmail(email) {
```

Comment:
This could be removed so we can just use userId and have an association in the user's model instead